src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -170,6 +170,8 @@
     <Compile Include="System\Resources\ResourceSet.cs" />
     <Compile Include="System\Runtime\InteropServices\CriticalHandle.cs" />
     <Compile Include="System\Text\CodePageDataItem.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.Unix.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.HillClimbing.Unix.cs" />
```

Comment:
We want the code to be structured such that it is possible to enable it on Windows. You have done a step towards that by renaming the files from `*.Unix.cs` to `*.cs`.

But we do not want to be compiling the code in by default on Windows, at least for now. Could you please add  `Condition="'$(TargetsWindows)'!='true'` on the new files in the .csproj file?